.github/CODEOWNERS

```diff
@@ -1,17 +1,27 @@
 # CODEOWNERS adds reviewers depending on the location of the change.
+
 # Each line is a file pattern followed by one or more owners.
+
 # Order is important; the last matching pattern takes the most
+
 # precedence. When someone opens a pull request that only
+
 # modifies JS files, only @js-owner and not the global
+
 # owner(s) will be requested for a review.
 
 # These owners will be the default owners for everything in
+
 # the repo. Unless a later match takes precedence,
+
 # @forcedotcom/pdt will be requested for
+
 # review when someone opens a pull request.
-*       @forcedotcom/pdt
+
+-       @forcedotcom/ide-experience
 
 # If the change occurs within the docs directory, add the doc maintainers.
+
```

Comment:
i am..not sure why it added a bunch of blank lines...I'll see if it's prettier or if I can revert.

Reply:
This file was defaulting to markdown which was why it was altering the spacing.  I just removed that association and it was good to go